func_sig_registry/utils/solidity.py

```diff
@@ -136,11 +137,23 @@ def to_canonical_type(value):
 
 
 def normalize_function_signature(raw_signature):
-    fn_name = extract_function_name(raw_signature)
-    raw_arguments = re.findall(FUNCTION_ARGUMENT_TYPES_REGEX, raw_signature)
+    fn_name, args = extract_function_name(raw_signature)
+    raw_arguments = re.findall(FUNCTION_ARGUMENT_TYPES_REGEX, args)
 
     arguments = [
         "".join((to_canonical_type(t), sub))
         for t, sub in raw_arguments
     ]
     return "{fn_name}({fn_args})".format(fn_name=fn_name, fn_args=','.join(arguments))
+
+def test():
```

Comment:
Can you remove this and add a test for this in https://github.com/pipermerriam/ethereum-function-signature-registry/blob/master/tests/web/utility/test_normalize_function_signature.py